R/incidence.R

```diff
@@ -140,8 +146,10 @@ incidence.integer <- function(dates, interval = 1L, groups = NULL,
     first.date <- min(dates)
     last.date <- max(dates)
     interval <- as.integer(round(interval))
-    if (interval == 7L) {
-      first.date <- 0L
+    if ("iso_week" %in% names(dots)) {
+      if (interval == 7L & dots$iso_week == TRUE) {
```

Comment:
yes, && is appropriate for programming control-flow and typically preferred in if clauses. I will fix it.